plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/LocalCheInfrastructureProvisioner.java

```diff
@@ -89,7 +100,30 @@ public void provision(Environment envConfig, CheServicesEnvironmentImpl internal
                    .add(SystemInfo.isWindows() ? pathEscaper.escapePath(projectFolderVolume)
                                                : projectFolderVolume);
 
+        if (cheUserId != null) {
```

Comment:
Is it on purpose that [there](https://github.com/eclipse/che/pull/3376/files#diff-fb8552dea5c1316ac9fd0b0988377f77R307) you also check for emptiness and here you don't